tools/update_required_branch_checks.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Updates the requires status checks for a branch."""
+
+import argparse
+from github import Github
+from typing import List
+
+YOUR_GITHUB_TOKEN = ''
+
+TARGET_REPO = 'youtube/cobalt'
+
+EXCLUDED_CHECK_PATTERNS = ['feedback/copybara', '_on_device_', r'${{']
```

Comment:
Since I just merged https://github.com/youtube/cobalt/pull/693 this should also include the codecov checks: `codecov/patch` and `codecov/project`.

https://github.com/youtube/cobalt/pull/693/checks?check_run_id=14516031443
https://github.com/youtube/cobalt/pull/693/checks?check_run_id=14516031438